packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -128,6 +130,13 @@ export const EventTypeDescription = ({
               </Badge>
             </li>
           ) : null}
+          <li>
+            <Link href={`/insights?isAll=false&filter=event-type&eventTypeId=${eventType.id}`}>
```

Comment:
@eunjae-lee @sean-brydon this is currently not working consistently because the URL-param are borked. that PR needs to go live first